downloader/src/main/kotlin/VersionControlSystem.kt

```diff
@@ -128,6 +129,11 @@ abstract class VersionControlSystem {
                 workingTree.getInfo().copy(path = workingTree.getPathToRoot(path))
             } ?: VcsInfo.EMPTY
         }
+
+        /**
+         * Return glob patterns matching all potential license or patent files.
+         */
+        internal fun getLicenseFileGlobPatterns(): List<String> = ALL_LICENSE_FILENAMES.map { "**/$it" }
```

Comment:
Why is this `internal` instead of `protected`, as there are no tests for it?

Reply:
Because `protected`` is not allowed in the companion object.